src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+class BasicValidationServiceTest {
+
+    private final BasicValidationService basicValidationService = new BasicValidationService();
```

Comment:
it's better to create beforeEach test a new instance, so no information that was from previous test is saved in memmory

Reply:
Understood, I think it would be better if I did as you said.